doc/GoogleTranslateConnector.md

```diff
@@ -0,0 +1,67 @@
+## Reshuffle Google Translate Connector
+
+`npm install reshuffle-google-connectors`
+
+_ES6 import_: `import { GoogleTranslateConnector } from 'reshuffle-google-connectors'` 
+
+This is a [Reshuffle](https://dev.reshuffle.com) connector that provides an Interface to the Google Translate Client.
+
+#### Setup
+You need a project in Google Cloud that has the Cloud Translation API enabled and credentials to make authenticated calls. 
+See more details [here](https://cloud.google.com/translate/docs/setup)
+
+
+#### Configuration Options:
+```typescript
+interface GoogleTranslateConnectorConfigOptions {
+  credentials: string
+}
+```
+#### Connector events
+N/A
+
+#### Connector actions
+
+##### translateText
+Returns the translated text response from source to target language
+
+```typescript
+translate(
+    text: string,
+    projectId: string, // Google Cloud Project Id, see https://cloud.google.com/resource-manager/docs/creating-managing-projects
+    location: string,  // Project's location like 'global' or 'us-central1'
+    mimeType: string,  // mime types: text/plain, text/html
+    source: string,    // e.g. 'en',
+    target: string,    // e.g. 'fr'
+  ): Promise<google.cloud.translation.v3.TranslateTextResponse>
+```
+
+###### Example
+```js
+  const connector = new GoogleTranslateConnector(app, null)
+  const result = await connector.translate(['Hello world', 'Text to translate'], 'project-01', 'global', 'text/plain', 'en', 'fr')
```

Comment:
connector.translate function takes a string as text but you pass an array here

Reply:
Added string[] option

Reply:
bonus feature: it could be nice to accept both string and string[] here. And return a translation or a array of translation